### DISCUSSION TITLE
为什么我宝塔安装了docker，查看也在运行，我反向代理了，访问就是提示502呢，直接访问IP和端口也不行

### DISCUSSION THREAD

**Discussion by haoqicunzhang** [lobehub/lobehub#1843] [Self Hosting]
![1](https://github.com/lobehub/.github/assets/55733532/72c0de5a-4968-4154-8ffb-b50cbafdc212)

![1](https://github.com/lobehub/.github/assets/55733532/141a00c4-e814-4bfa-a617-26fcda15b423)
我在本地电脑搭建就能访问，搭建在宝塔的DOCKER里搭建了好几次都不行

**Comment by MapleEve** [comment]
先确认
1. 端口和防火墙是否打开
2. 到宝塔本机去访问3210端口看看通不通

**Comment by arvinxx** [comment]
502 bad gateway 可能是 Nginx 转发到 docker 的路径出了问题，这个还是需要你自己排查的
